src/csg_tree.cpp

```diff
@@ -111,29 +111,29 @@ std::shared_ptr<CsgNode> CsgNode::Boolean(
 }
 
 std::shared_ptr<CsgNode> CsgNode::Translate(const vec3 &t) const {
-  mat4x3 transform(1.0);
+  mat3x4 transform(1.0);
```

Comment:
This is not the identity matrix, this is a matrix with all entries = 1.

Reply:
Good catch - I had fixed a bunch of those, but must have missed this one.